proto/cosmos/protocolpool/v1/types.proto

```diff
@@ -30,4 +30,27 @@ message Budget {
   // For example, if a period is set to 3600, it represents an action that
   // should occur every hour (3600 seconds).
   google.protobuf.Duration period = 8 [(gogoproto.stdduration) = true];
-}
\ No newline at end of file
+}
+
+// ContinuousFund defines the fields of continuous fund proposal.
+message ContinuousFund {
+  // Title is the title of the funds.
+  string title = 1;
+  // Description of the funds.
+  string description = 2;
+  // Recipient address of the account receiving funds.
+  string recipient = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Metadata is any arbitrary metadata attached.
+  string metadata = 4;
```

Comment:
What is title, description and metadata are for actually?
Same for all messages from protocol pool.